Application.cpp

```diff
@@ -343,13 +345,16 @@ void Application::buildSwapChain() {
 	glfwGetFramebufferSize(m_window, &width, &height);
 
 	std::cout << "Creating swapchain..." << std::endl;
-	m_swapChainDesc = {};
+	m_swapChainDesc = wgpu::SwapChainDescriptor{};
 	m_swapChainDesc.width = (uint32_t)width;
 	m_swapChainDesc.height = (uint32_t)height;
-	m_swapChainDesc.usage = TextureUsage::RenderAttachment | TextureUsage::TextureBinding;
+	m_swapChainDesc.usage = TextureUsage::RenderAttachment;
 	m_swapChainDesc.format = m_swapChainFormat;
 	m_swapChainDesc.presentMode = PresentMode::Fifo;
 	m_swapChain = m_device.createSwapChain(m_surface, m_swapChainDesc);
+	if (!m_swapChain) {
+		throw std::runtime_error("Failed to create swapchain");
+	}
```

Comment:
For the sake of consistency, either add this for all required objects that might be null or don't add this and just look at the cout when debugging

Reply:
Which do you prefer? For me, either would be fine.

Reply:
Remove it, otherwise I'll have to update the 100 branches to follow teh same convention ^^'